Sources/NIOHTTPClient/Utils.swift

```diff
@@ -17,38 +17,59 @@ import NIOHTTP1
 
 public class HandlingHTTPResponseDelegate<T>: HTTPClientResponseDelegate {
     struct EmptyEndHandlerError: Error {}
-
+    
     public typealias Result = T
-
+    
     var handleHead: ((HTTPResponseHead) -> Void)?
     var handleBody: ((ByteBuffer) -> Void)?
     var handleError: ((Error) -> Void)?
     var handleEnd: (() throws -> T)?
-
+    
     public func didTransmitRequestBody(task: HTTPClient.Task<T>) {}
-
+    
     public func didReceiveHead(task: HTTPClient.Task<T>, _ head: HTTPResponseHead) {
         if let handler = handleHead {
             handler(head)
         }
     }
-
+    
     public func didReceivePart(task: HTTPClient.Task<T>, _ buffer: ByteBuffer) {
         if let handler = handleBody {
             handler(buffer)
         }
     }
-
+    
     public func didReceiveError(task: HTTPClient.Task<T>, _ error: Error) {
         if let handler = handleError {
             handler(error)
         }
     }
-
+    
     public func didFinishRequest(task: HTTPClient.Task<T>) throws -> T {
         if let handler = handleEnd {
             return try handler()
         }
         throw EmptyEndHandlerError()
     }
 }
+
+extension ClientBootstrap {
+    static func makeHTPPClientBootstrapBase(group: EventLoopGroup, request: HTTPClient.Request, configuration: HTTPClient.Configuration, channelInitializer: @escaping (Channel)->EventLoopFuture<Void>) -> ClientBootstrap {
+        return ClientBootstrap(group: group)
+            .channelOption(ChannelOptions.socket(SocketOptionLevel(IPPROTO_TCP), TCP_NODELAY), value: 1)
+            .channelInitializer { channel in
+                // let encoder = HTTPRequestEncoder()
+                // let decoder = ByteToMessageHandler(HTTPResponseDecoder(leftOverBytesStrategy: .forwardBytes))
```

Comment:
What're these here for?

Reply:
Oops, forgot to definitively remove it after I made this HTTP version agnostic